.gitignore

```diff
@@ -41,4 +41,6 @@ yarn-debug.log*
 .yarn-integrity
 
 config/database.yml
-/config/local_env.yml
\ No newline at end of file
+/config/local_env.yml
+/config/master.key
+config/credentials.yml.enc
```

Comment:
do not include the `config/credentials.yml.enc` to `.gitignore`

Reply:
Understood. Fixed